content/pages/sprint/2023/10-hsinchu.rst

```diff
@@ -0,0 +1,205 @@
+----------------------
+
+To join the sprint, please bring your laptop and `sign up <#sign-up>`__.  You are also 
+very welcome to bring your project. Please `contact us <#contact-us>`__ if you have any 
+questions.
+
+Scisprint, hosted by the sciwork community, is a monthly coding sprint. It would like to 
+facilitate discussions and exchanges among people in the fields of science, numerical 
+computation, and engineering. Participants, regardless of experience level, can gain valuable 
+development insights in this event.
+
+This event includes a `hacking session <#hacking-session>`__ and `career conversation <#career-conversation>`__.
+
+We would like to provide a supportive and friendly environment for all attendees to support more developers
+to join in the open-source communities. 
+
+Hacking Session
+------------------
+
+It aims to encourage collaboration and interaction among developers through project 
+participation. The projects cover various fields, including but not limited to science, 
+numerical computation, and engineering. You are also encouraged to share your own projects 
+in scisprint. Refer to `project list <#project-list>`__ below for more details.
+
+Project List
++++++++++++++
+
+modmesh
+^^^^^^^^
+
+Toolkit for solving partial differential equations and analysis of the results.
+
+- **Related Subjects:** Python, C++, PDE
+- **Project Link:** `Github <https://github.com/solvcon/modmesh>`__
+- **Project Contact:** Yung-Yu Chen (discord: @yyc#7718)
+
+|
+
+sciwork portal
+^^^^^^^^^^^^^^^
+
+Sciwork Portal is a project for maintaining our official website – Sciwork.dev, which was 
+built by Pelican with tailwindCSS, and deployed by Netfliy.
+
+- **Project Link:** `Github <https://github.com/sciwork/swportal>`__
+- **Project Contact:** Aya yu (discord: @tN#6722), Chester (discord: @chester)
+
+|
+
+pyLiteracy
+^^^^^^^^^^^
+
+Mandarin LocativePeriodic-zai [在] and Repetitive-zai [再] parser based on syntactical 
+context. 
+
+- **Project Link:** `Github <https://github.com/Chenct-jonathan/Loc_zai_and_Rep_zai_parser>`__
+- **Project Contact:** Jonathan Chen (discord: @陳畯田), PeterWolf (discord: @PeterWolf#1422)
+
+|
+
+uTensor
+^^^^^^^^
+
+uTensor is an extremely lightweight machine learning inference framework built on TensorFlow 
+and optimized for Arm targets. It consists of a runtime library and an offline tool that handles 
+most of the model translation work. This repo holds the core runtime and some example implementations 
+of operators, memory managers/schedulers, and more, and the size of the core runtime is only ~2KB!
+
+- **Project Link:** `Github <https://github.com/uTensor/uTensor>`__
+- **Project Contact:** Dboy(discord: @dboyliao#1295)
+
+
+|
+
+pydoccht
+^^^^^^^^^
+
+The Traditional Chinese (zh_TW) translation project of the Python official documents.
+
+- **Project Link:** `Github <https://github.com/python/python-docs-zh-tw>`__
+- **Project Contact:** Matt Wang (@mattwang44), Josix (@Josix#3800)
+
+|
+
+Cytnx
```

Comment:
Cytnx won't participate in the October event.

Reply:
OK. I deleted this.